### ISSUE TITLE
here link is not opening in a new tab.

### ISSUE THREAD

**Issue by benjaminthedev** [FreeCodeCamp/FreeCodeCamp#12337]
Challenge [Get Geolocation Data](https://www.freecodecamp.com/challenges/get-geolocation-data#?solution=%0Afccss%0A%20%20%2F%2F%20Only%20change%20code%20below%20this%20line.%0A%20%20%0A%20%20if%20(navigator.geolocation)%20%7B%0A%20%20navigator.geolocation.getCurrentPosition(function(position)%20%7B%0A%20%20%20%20%24(%22%23data%22).html(%22latitude%3A%20%22%20%2B%20position.coords.latitude%20%2B%20%22%3Cbr%3Elongitude%3A%20%22%20%2B%20position.coords.longitude)%3B%0A%20%20%7D)%3B%0A%7D%0A%20%20%0A%20%20%2F%2F%20Only%20change%20code%20above%20this%20line.%0Afcces%0A%3Cdiv%20id%20%3D%20%22data%22%3E%0A%20%20%3Ch4%3EYou%20are%20here%3A%3C%2Fh4%3E%0A%20%20%0A%3C%2Fdiv%3E%0A) has an issue.
User Agent is: <code>Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/55.0.2883.75 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```html

<script>
  // Only change code below this line.
  
  if (navigator.geolocation) {
  navigator.geolocation.getCurrentPosition(function(position) {
    $("#data").html("latitude: " + position.coords.latitude + "<br>longitude: " + position.coords.longitude);
  });
}
  
  // Only change code above this line.
</script>
<div id = "data">
  <h4>You are here:</h4>
  
</div>

```

In the instructions at the bottom it says:

"Don't forget to use double quotes around selectors as mentioned here."

It should open in a new link - that is all.

**Comment by dhcodes** [comment]
I'm fine with this. First-timers only.

Change [this line](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/5b14f1631b46d3f8e02b479c18a353e8669b96b9/seed/challenges/01-front-end-development-certification/json-apis-and-ajax.json#L490) to open the link in a new tab/window by adding `target="_blank"` to the anchor link.

Please follow the [contribution guidelines](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md) when making your pull request.

**Comment by tmashuang** [comment]
@dhcodes I would like to work on this issue as a first time contributor please.

**Comment by baoqchau** [comment]
Hello @dhcodes . I tried to change as you suggested but it still open in the same tab. I tried it with Chromium 55 and Firefox 50.0.1 Should we change it to something like `onclick="window.open` instead?
